ion-schema/src/isl/isl_constraint.rs

```diff
@@ -1372,54 +1298,28 @@ pub struct IslValidValuesConstraint {
 
 impl IslValidValuesConstraint {
     /// Provides a way to programmatically construct valid_values constraint
-    /// Returns IonSchemaError whenever annotations are provided within ValidValue::Element
-    /// only `range` annotations are accepted for ValidValue::Element
-    pub fn new(valid_values: Vec<ValidValue>, isl_version: IslVersion) -> IonSchemaResult<Self> {
-        let valid_values: IonSchemaResult<Vec<ValidValue>> = valid_values
-            .iter()
-            .map(|v| match v {
-                ValidValue::Range(r) => Ok(v.to_owned()),
-                ValidValue::Element(e) => ValidValue::from_ion_element(e, isl_version),
-            })
-            .collect();
-        Ok(Self {
-            valid_values: valid_values?,
-        })
```

Comment:
🗺️ `ValidValue::Element` can no longer have annotations, so all of this logic is no longer necessary.